styles.css

```diff
@@ -308,19 +310,15 @@ body {
     cursor: pointer;
     z-index: 10;
     -webkit-transition: .05s linear all;
-        -moz-transform: .05s linear all;
```

Comment:
i deteted those because my code editor was giving `Mismatched property value` error and idk what to do about it